controllers/permits/applications/special-uses/commercial/temp-outfitters/index.js

```diff
@@ -119,103 +118,166 @@ function fileErrors(missingFiles){
 	output = output.trim();
 	return output;
 }
+
+function concatErrors(errorMessages){
```

Comment:
@cameronwolf concatErrors moved to utility, and buildErrorMessage hasn't updates though